src/components/ComboBox/MultiComboBox.tsx

```diff
@@ -307,7 +307,15 @@ export function MultiComboBox<T>({
       </InputArea>
 
       <Suffix themes={theme}>
-        <FaCaretDownIcon color={isFocused ? theme.color.TEXT_BLACK : theme.color.BORDER} />
+        <FaCaretDownIcon
+          color={
+            !disabled
+              ? isFocused
+                ? theme.color.TEXT_BLACK
+                : theme.color.TEXT_GREY
+              : theme.color.BORDER
```

Comment:
条件を否定で始めないほうが読みやすいと思います。
また、`color.BORDER` もよくわからないので `TEXT_DISABLED` を使いたいです。

SingleComboBox も同じ。

```suggestion
            disabled
              ? theme.color.TEXT_DISABLED
              : isFocused
              ? theme.color.TEXT_BLACK
              : theme.color.TEXT_GREY
```